tests/manage/test_rbd_csi_default_sc.py

```diff
@@ -0,0 +1,94 @@
+"""
+Basic test for creating PVC with default StorageClass - RBD-CSI
+"""
+
+import logging
+import pytest
+import base64
+from ocs import defaults
+from ocsci.config import ENV_DATA
+from ocsci.testlib import tier1, ManageTest
+from resources.ocs import OCS
+from resources import pod
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This is a test fixture
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    global SECRET
+    self.rbd_secret = defaults.CSI_RBD_SECRET.copy()
+    del self.rbd_secret['data']['kubernetes']
+    self.rbd_secret['data']['admin'] = get_admin_key_from_ceph_tools()
+    SECRET = OCS(**self.rbd_secret)
+    SECRET.create()
+
+
+def teardown():
+    """
+    Tearing down the environment
+    """
+    log.info("Deleting PVC")
+    PVC.delete()
+    log.info("Deleting StorageClass")
+    STORAGE_CLASS.delete()
+    log.info("Deleting Secret")
+    SECRET.delete()
+
+
+def get_admin_key_from_ceph_tools():
+    """
```

Comment:
this is already implemented in helpers.get_admin_key()

Reply:
Done